homeassistant/components/duckdns/translations/en.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file is no longer needed. Please remove it from the PR.
This also hightlights that strings.json is missing...